include/tvm/runtime/device_api.h

```diff
@@ -268,6 +268,8 @@ inline const char* DeviceName(int type) {
       return "webgpu";
     case kDLHexagon:
       return "hexagon";
+    case 129:
```

Comment:
so this is an RPC device ID. you shouldn't need to define this, and we can't submit this chance to device_api.h (only DeviceType enums should be here

Reply:
yeah, sorry, I left it while the PR is in Draft state. I'll need to fix it in a follow-on PR tho (I still need to figure out what's exactly missing - fwiw it seems that `module.benchmark()` and so `module.time_evaluator` works fine on micro targets with that change hence it seems not much is missing to make it work the right way). For now I can add `--print-time` to the list of non-supported options for micro targets on `tvmc run`.